utils/dataset_utils.py

```diff
@@ -68,6 +72,67 @@
 
 _MAX_ROWS = 200000
 
+def initialize_cache_hub_repo(cache_path, dataset_cache_dir):
+    """
+    This function tries to initialize a dataset cache on the huggingface hub. The
+    function expects you to have HUB_CACHE_ORGANIZATION=<the organization you've set up on the hub to store your cache>
+    and HF_TOKEN=<your hf token> on separate lines in a file named .env at the root of this repo.
+
+    Args:
+        cache_path (string):
+            The path to the local dataset cache.
+        dataset_cache_dir (string):
+            The name of the dataset repo on the huggingface hub that you want.
+    """
+
+    if Path(".env").is_file():
```

Comment:
I would move the following into its own helper function, and then call it in the 2 functions that use it but repeat the same thing. E.g.,

```
def _load_env():
    if Path(".env").is_file():
        load_dotenv(".env")

    HF_TOKEN = getenv("HF_TOKEN")
    HUB_CACHE_ORGANIZATION = getenv("HUB_CACHE_ORGANIZATION")
```